From a1cc54aaa09853015d3130cae6cfb5273b6ec95d Mon Sep 17 00:00:00 2001
From: leftieFriele <leftie.friele@gmail.com>
Date: Wed, 2 Oct 2013 13:51:38 +0200
Subject: [PATCH] initial commit

---
 src/style/insertCss.js | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/src/style/insertCss.js b/src/style/insertCss.js
new file mode 100644
index 0000000..b7b9671
--- /dev/null
+++ b/src/style/insertCss.js
@@ -0,0 +1,12 @@
+module.exports = function(css) {
+    var head = document.getElementsByTagName('head')[0];
+    var style = document.createElement('style');
+    style.type = 'text/css';
+    if (style.styleSheet) {
+        style.styleSheet.cssText = css;
+    } else {
+        style.appendChild(document.createTextNode(css));
+    }
+
+    head.appendChild(style);
+};
